tools/distpartitioning/utils.py

```diff
@@ -387,6 +381,8 @@ def get_idranges(names, counts):
         list of node/edge types as strings
     counts : list of lists
         each list contains no. of nodes/edges in a given chunk
+    expected_num_chunks : expected number of returned chunks
+        chunks are collapsed according to expected number of chunks
```

Comment:
- What goes after the colon symbol `:` should be the type of the argument.
- The description does not make sense to me. Please reword.